components/ciphertool/src/main/java/org/wso2/ciphertool/utils/Utils.java

```diff
@@ -414,6 +495,51 @@ public static Map<String, String> getSecreteFromConfiguration(String configFileP
         return context;
     }
 
+    /**
+     * Read from toml file and return keystore data.
+     *
+     * @param configFilePath    file path to deployment toml
+     * @param keystoreName      name of the keystore
+     * @return      Map of keystore data
+     */
+    public static Map<String, String> getKeystoreFromConfiguration(String configFilePath, String keystoreName) {
+        Map<String, String> context = new LinkedHashMap<>();
+        try {
+            TomlParseResult result = Toml.parse(Paths.get(configFilePath));
+            if (result.hasErrors()) {
+                throw new CipherToolException("Error while parsing TOML config file");
+            }
+            TomlTable table = result.getTable(keystoreName);
+            if (table != null) {
+                table.dottedKeySet().forEach(key -> context.put(key, table.getString(key)));
+            }
+
+        } catch (IOException e) {
+            System.out.println("Error parsing file " + configFilePath + e.toString());

```

Comment:
Dont use System.out to print error messages. lets use logger

Reply:
Ciphertool does not have a logger as a dependency

Reply:
Since it is an interactive tool, most of the lines should be written back to the command prompt for the user to interact with.